server/src/main/java/org/apache/druid/server/coordinator/rules/ForeverBroadcastDistributionRule.java

```diff
@@ -56,6 +56,12 @@ public boolean appliesTo(Interval interval, DateTime referenceTimestamp)
     return true;
   }
 
+  @Override
+  public Interval getInterval(DateTime referenceTimestamp)
+  {
+    return Rules.FOREVER_INTERVAL;
```

Comment:
Better use `Intervals.ETERNITY`.

Reply:
Updated. Fwiw, the initial version had a slightly different logic and that was causing validation to fail in some cases because [eternity](https://github.com/apache/druid/blob/master/processing/src/main/java/org/apache/druid/java/util/common/JodaUtils.java#L44) is `[Long.MIN_VALUE / 2, Long.MAX_VALUE / 2]`.